nextflow_schema.json

```diff
@@ -24,7 +24,7 @@
                     "default": "rnaseq",
                     "description": "A string identifying the technology used to produce the data",
                     "help_text": "Currently 'rnaseq' or 'affy_array' may be specified.",
-                    "enum": ["rnaseq", "affy_array"],
+                    "enum": ["rnaseq", "affy_array", "non_affy_array"],
```

Comment:
I'm not convinced this is the most helpful renaming. e.g. we may at some point have a module that deals with illumina-style arrays. Should this not be something that indicates the pulling of soft files from GEO?

Reply:
Agreed, sorry for the lack of inspiration :) I replaced with geo_soft_file. Let me know if that's any better

Reply:
looks good